sys-process/oomd/oomd-0.5.0-r1.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+SRC_URI="https://github.com/facebookincubator/oomd/releases/tag/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI="https://github.com/facebookincubator/oomd/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64"
+
+inherit meson
+
+DESCRIPTION="oomd is userspace Out-Of-Memory (OOM) killer for linux systems."
+HOMEPAGE="https://github.com/facebookincubator/oomd"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="man systemd"
+DEPEND="dev-libs/jsoncpp:=
+	systemd? ( sys-apps/systemd )"
+RDEPEND="${DEPEND}"
+CONFIG_CHECK="~CONFIG_PSI"
+ERROR_PSI="CONFIG_PSI is required for oomd to function"
+
+src_configure() {
+	local emesonargs
+	emesonargs=(
+		$(meson_feature man)
+		$(meson_feature systemd)
+		)
+	meson_src_configure
+}
+
+src_install() {
+	meson_src_install
+	einstalldocs
+	doman "${S}"/man/oomd.1
```

Comment:
It's mildly interesting that meson isn't installing both the docs and the man page for us. Might be worth seeing if we can fix that

Reply:
Yeah, it works without it!